pkg/archive/archive.go

```diff
@@ -405,6 +405,8 @@ func Tar(path string, compression Compression) (io.ReadCloser, error) {
 // paths are included in `options.IncludeFiles` (if non-nil) or not in `options.ExcludePatterns`.
 func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error) {
 
+	srcPath = fixVolumePathPrefix(srcPath)
```

Comment:
@jhowardmsft this means on windows `fixVolumePathPrefix` will always return `\\?\`+srcPath
Shouldn't that be reserved for "volumes" (sorry i'm confused by volumes and `\\?\`)